samples/matter/common/src/bridge/ble_connectivity_manager.cpp

```diff
@@ -719,6 +743,19 @@ CHIP_ERROR BLEConnectivityManager::Connect(BLEBridgedDeviceProvider *provider, C
 
 	bt_conn *conn{};
 	bt_addr_le_t btAddress = provider->GetBtAddress();
+
+#ifdef CONFIG_BRIDGE_SET_BT_CONNECTION_PARAMS
+	struct bt_le_conn_param connParams = {
+		.interval_min = CONFIG_BRIDGE_BT_CONNECTION_INTERVAL_MIN,
+		.interval_max = CONFIG_BRIDGE_BT_CONNECTION_INTERVAL_MAX,
+		.latency = CONFIG_BRIDGE_BT_CONNECTION_LATENCY,
+		.timeout = CONFIG_BRIDGE_BT_CONNECTION_TIMEOUT,
+	};
+
+	int err = bt_conn_le_create(&provider->GetBLEBridgedDevice().mAddr, create_param, &connParams, &conn);
```

Comment:
We could do something similar to the `ParamChangeRequestHandler` and see if params returned by `GetScannedDeviceConnParams` fit our range or not. If the peripheral range is ok for our bridge, we can accept it. Otherwise, let's use our parameters.